src/test/TEST_ij/matrix.saved.sunspot

```diff
@@ -24,11 +24,11 @@ Final Relative Residual Norm = 4.233149e-01
 
 # Output file: matrix.out.6
 Iterations = 5
-Final Relative Residual Norm = 4.233149e-01
+Final Relative Residual Norm = 3.515462e-01
 
 # Output file: matrix.out.7
 Iterations = 5
-Final Relative Residual Norm = 4.233149e-01
+Final Relative Residual Norm = 3.515462e-01
```

Comment:
These changes seem large.  Is this expected?

Reply:
This is due to a bug fix in IJ matrix assembly for sycl:
https://github.com/hypre-space/hypre/pull/904/files#diff-7ddf8e4e5bf557900adda9b6a7bf028a61225dbbbc09b2018c575174aa717470L304
I previously had some arguments reversed... I guess that the correctness of the IJ matrix assembly is not tested in the regressions? If you run the ij driver with -test_ij, I guess it just tests that the code runs, not necessarily that you get the correct result, since I had missed this bug up to now.

Reply:
Yes, good catch. I should add some regression tests. We can use ij.c or ij_assembly.c that is just for assembly. Thanks!